Manual.md

```diff
@@ -76,17 +76,19 @@ packages for XBPS, the `Void Linux` native packaging system.
 <a id="Introduction"></a>
 ## Introduction
 
-The `void-packages` repository contains all `source` packages that are the
-recipes to download, compile and build binary packages for `Void`.
-Those `source` package files are called `templates`.
+The `void-packages` repository contains all the
+recipes to download, compile and build binary packages for Void Linux.
+These `source` package files are called `templates`.
 
-The `template files` are `GNU bash` shell scripts that must define some required/optional
+The `template' files are `GNU bash` scripts that define some optional or required
 `variables` and `functions` that are processed by `xbps-src` (the package builder)
 to generate the resulting binary packages.
 
-By convention, all templates start with a comment briefly explaining what they
-are. In addition, pkgname and version can't have any characters in them that
-would require them to be quoted, so they are not quoted.
+By convention, all templates start with a comment saying that it is a
+`template file` for a certain package. In addition, `pkgname` and `version` can't
+contain any characters in that would require them to be quoted, so they are not quoted.
```

Comment:
This fits better into description of variables, as there are more restictions on version.